deploy/example_clusters/cr-cluster-endpoint-wrapper.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: "kubedirector.hpe.com/v1beta1"
+kind: "KubeDirectorCluster"
+metadata: 
+  name: "deploymt-engine"
+
+spec:
+  app: flask-and-inf-eng
+  #connections:
+    #models:
```

Comment:
Let's confirm with UI team/Nabin on how is UI expecting "connections" to be

Reply:
The connections template is now being added to every App from the UI. Removing this field from the default yaml